doc/rst/source/reference/map-projections.rst

```diff
@@ -1110,8 +1110,8 @@ range of the plot, specified by the (**-R**) option.
 
 .. _-Jh:
 
-Hammer projection (**-Jh** **-JH**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Hammer equal-area projection (**-Jh** **-JH**)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
Hammer projection (**-Jh** **-JH**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
For this change please see commit https://github.com/GenericMappingTools/gmt/pull/8567/commits/dcc4be23c7df26b645927bdc70a928ac03e7f153.